_posts/2018-08-19-openfaas-on-fargate.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Faas on Fargate&#58; run OpenFaaS on AWS Fargate
+description: Edward Wilde introduces running OpenFaaS on AWS Fargate - a way to deploy and run functions without managing servers
+date: 2018-08-02
+image: /images/openfaas-on-fargate/astronomy-atmosphere-earth-220201.jpg
+categories:
+  - fargate
+author_staff_member: ed
+---
+## Why run OpenFaaS on AWS Fargate
+One of the great things about OpenFaaS is it's portability. You can run OpenFaaS out of the box using 
+officially supported platforms like are Kubernetes and Docker Swarm . The [community](https://github.com/openfaas/faas/blob/master/community.md) 
+has added other platforms that allow you to run OpenFaaS in more places. A good example of this is deploying OpenFaaS
+on Hashicorp's [nomad](https://www.hashicorp.com/blog/functions-as-a-service-with-nomad) application scheduler.
+
+### Complexity
```

Comment:
`Complexity` is quite negative, Kubernetes etc are our core target audience, we don't want to put people off "OpenFaaS"